components/esp_lvgl_port/src/lvgl8/esp_lvgl_port_disp.c

```diff
@@ -239,7 +247,14 @@ static lv_disp_t *lvgl_port_add_disp_priv(const lvgl_port_display_cfg_t *disp_cf
 #if CONFIG_IDF_TARGET_ESP32S3 && ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 0, 0)
         buffer_size = disp_cfg->hres * disp_cfg->vres;
         ESP_GOTO_ON_ERROR(esp_lcd_rgb_panel_get_frame_buffer(disp_cfg->panel_handle, 2, (void *)&buf1, (void *)&buf2), err, TAG, "Get RGB buffers failed");
+#elif CONFIG_IDF_TARGET_ESP32P4 && ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 3, 0)
+        buffer_size = disp_cfg->hres * disp_cfg->vres;
+        ESP_GOTO_ON_ERROR(esp_lcd_dpi_panel_get_frame_buffer(disp_cfg->panel_handle, 2, (void *)&buf1, (void *)&buf2), err, TAG, "Get RGB buffers failed");
 #endif
+
+        trans_sem = xSemaphoreCreateCounting(1, 0);
```

Comment:
It seems, we forgot delete semaphore in `lvgl_port_remove_disp`. Same for LVGL8 and LVGL9.

Reply:
Yes, that was an oversight.

Reply:
Please, could you update it in `components/esp_lvgl_port/src/lvgl9/esp_lvgl_port_disp.c` too?

Reply:
Oh, I forgot. Alright, I will make changes later.

Reply:
Thanks.